ui/config/stage.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  apiPrefix: "https://stage.cobudget.co/api/v1"
+}
```

Comment:
there's a `ui/config/staging.js` which has this in it: 

```js
module.exports = {
  apiPrefix: "https://api.cobudget.co/api/v1"
}
```

Feels like this is some duplication which needs resolving?

Reply:
@mixmix True. The only 3 configs that are actually in use is `default.js` (or `development.js`), `stage.js` and `production.js`. The reason for the duplication is a recent move in server setup. Feel free to delete `staging.js` - and and let me know if you need more details.